Userland/Libraries/LibGfx/AntiAliasingPainter.cpp

```diff
@@ -123,8 +123,52 @@ void Gfx::AntiAliasingPainter::draw_aliased_line(FloatPoint const& actual_from,
     draw_anti_aliased_line<AntiAliasPolicy::OnlyEnds>(actual_from, actual_to, color, thickness, style, alternate_color);
 }
 
+void Gfx::AntiAliasingPainter::draw_dotted_line(IntPoint point1, IntPoint point2, Gfx::Color color, int thickness)
```

Comment:
Why on earth is there no Gfx namespace in this file? Having to use these namespace prefixes within LibGfx itself is sad

Reply:
Huh, no idea, I'll move it into one now